code/game/objects/items/devices/chameleon_counter.dm

```diff
@@ -0,0 +1,77 @@
+/obj/item/chameleon_counterfeiter
+	name = "chameleon counterfeiter"
+	icon = 'icons/obj/device.dmi'
+	icon_state = "cham_counter"
+	flags = CONDUCT
+	slot_flags = SLOT_BELT
+	item_state = "electronic"
+	w_class = WEIGHT_CLASS_SMALL
+	origin_tech = "syndicate=3;magnets=3"
+	var/can_use = TRUE
+	var/saved_item
+	var/saved_icon
+	var/saved_icon_state
+	var/saved_item_state
+	var/saved_overlays
+	var/saved_underlays
+	var/activate_dummy = FALSE
+
+/obj/item/chameleon_counterfeiter/examine(mob/user)
+	. = ..()
+	if(activate_dummy == TRUE)
```

Comment:
```suggestion
	if(activate_dummy)
```
you will likely never need to do a hard TRUE check for variables that only store bools. 

writing `if(activate_dummy)` will already execute to `if(TRUE)` if activate dummy is true and it will pass the if check